### PR TITLE
Add edit creds corrective action to the explorer

### PR DESCRIPTION
**Pull request by abrooksv**
## Screenshots (if appropriate)

Downside is the action may show in 3 places but maybe thats okay
Screen Shot 2020-08-14 at 1 58 54 PM

Screen Shot 2020-08-14 at 1 58 42 PM

## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] Local run of `gradlew check` succeeds
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] A short description of the change has been added to the **CHANGELOG**

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by kiiadi** [comment]
I like having it as an explorer action - do we need it twice in the menu tho?

**Comment by abrooksv** [comment]
> I like having it as an explorer action - do we need it twice in the menu tho?

Thats what I was referring to. I can split corrective actions into explorer actions and status bar actions.

Thoughts @kiiadi ?

**Comment by kiiadi** [comment]
I see the corrective actions are in both the menu and on the explorer pane... are they in both menus? It feels a bit strange that 'credentials' is the catch-all place for corrective actions...

**Comment by abrooksv** [comment]
> I see the corrective actions are in both the menu and on the explorer pane... are they in both menus? It feels a bit strange that 'credentials' is the catch-all place for corrective actions...

Corrective actions show in the top level of the cred selector, it just also we hard coded to show Edit creds in the All Credentials selector

**Comment by abrooksv** [comment]
Screen shot showing multiple actions in explorer:
Screen Shot 2020-08-14 at 2 25 20 PM

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ChangeConnectionSettingsMenu.kt [hunk 1]
```diff
@@ -59,22 +59,27 @@ class ChangeAccountSettingsActionGroup(project: Project, private val mode: Chang
...
+                allCredentials.add(ActionManager.getInstance().getAction("aws.settings.upsertCredentials"))
+
+                actions.add(allCredentials)
             }
         }
 
-        actions.add(Separator.create())
-        actions.addAll(accountSettingsManager.connectionState.actions)
+        // Both mode == status bar version
+        if (mode == BOTH) {
```
Comment (kiiadi) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ChangeConnectionSettingsMenu.kt:79]: Should prly rename this mode at some point - cos you're right it is just the "StatusBar" version.

**Pull request merged**